.eslintrc

```diff
@@ -151,7 +151,7 @@
       "pageYOffset", "parent", "print", "removeEventListener", "resizeBy",
       "resizeTo", "screen", "screenLeft", "screenTop", "screenX", "screenY",
       "scroll", "scrollbars", "scrollBy", "scrollTo", "scrollX", "scrollY",
-      "self", "sessionStorage", "status", "statusbar", "stop",
```

Comment:
Is this change fine, or should I just use a pragma where it is used?
sessionStorage is preferred to localStorage for this use case as the former doesn't persist between tabs and sessions, while the latter does which could lead to issues.

Reply:
I think the idea here is to require use of $window.sessionStorage rather than just a naked sessionStorage... -- would that work?

(We ought to do the same thing for localStorage too, tbh.)

Reply:
Done added this back. 
For localStorage, that was already done (it was window.localStorage).